v2-migration/src/main/java/software/amazon/awssdk/v2migration/internal/utils/NamingConversionUtils.java

```diff
@@ -31,11 +31,13 @@ public final class NamingConversionUtils {
 
     static {
         PACKAGE_MAPPING.put("appregistry", "servicecatalogappregistry");
+        PACKAGE_MAPPING.put("augmentedairuntime","sagemakera2iruntime");
         PACKAGE_MAPPING.put("certificatemanager", "acm");
         PACKAGE_MAPPING.put("cloudcontrolapi", "cloudcontrol");
         PACKAGE_MAPPING.put("cloudsearchv2", "cloudsearch");
         PACKAGE_MAPPING.put("cloudwatchevidently", "evidently");
         PACKAGE_MAPPING.put("logs", "cloudwatchlogs");
+        PACKAGE_MAPPING.put("metrics","cloudwatch");
```

Comment:
Is `metrics` the package name for client side metrics? If so, the correct package is not cloudwatch. We don't support metrics yet in migration tool

Reply:
Yeah it is in the cloudwatchmetrics module
https://github.com/aws/aws-sdk-java/tree/master/aws-java-sdk-cloudwatchmetrics/src/main/java/com/amazonaws/metrics

Gotcha, will remove this mapping